integration/containerd/confidential/fixtures/agent-configuration-no-exec.toml

```diff
@@ -44,3 +44,6 @@ allowed = [
         "WaitProcessRequest",
         "WriteStreamRequest"
 ]
+
+# TODO - we can remove this once this file doesn't override cmdline parameters
+enable_signature_verification = false
```

Comment:
I not sure whether we need add `enable_signature_verification` at the beginning of the config file? I tried to add `https_proxy` before, and did not take effect when added at the bottom.

Reply:
Thanks for the tip - I'll try updating that. And we should probably look into it more later!

Reply:
Just an update - now the ability for the kernel_params to override agent config is in I've been able to remove this all together :)